### DISCUSSION TITLE
Add "Download" button to File Library Browse

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3091] [Feature Requests]
An important part of a DAM that we're lacking right now.

This should support bulk (multiple items selected) downloads too... which would ideally be zipped.

Credit: @meaganhaynes

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
